README.md

```diff
@@ -66,49 +66,50 @@ $ cargo add thiserror --rename error
-    -B --build              Add crate as build dependency.
-    --optional              Add as an optional dependency (for use in features). This does not work
-                            for `dev-dependencies` or `build-dependencies`.
-    --target <target>       Add as dependency to the given target platform. This does not work
-                            for `dev-dependencies` or `build-dependencies`.
-
-Options:
-    --rename=<alias>        Rename the dependency to alias in Cargo.toml
-    --sort                  Keep dependencies sorted
-    --upgrade=<method>      Choose method of semantic version upgrade. Must be one of
-                            "none" (exact version), "patch" (`~` modifier), "minor"
-                            (`^` modifier, default), or "all" (`>=`).
-    --manifest-path=<path>  Path to the manifest to add a dependency to.
-    --allow-prerelease      Include prerelease versions when fetching from crates.io (e.g.
-                            '0.6.0-alpha'). Defaults to false.
-    -q --quiet              Do not print any output in case of success.
-    -h --help               Show this help page.
-    -V --version            Show version.
+cargo-add
+Add dependency to a Cargo.toml manifest file
+
+USAGE:
+    cargo add [FLAGS] [OPTIONS] <crate>...
+
+FLAGS:
+        --allow-prerelease       Include prerelease versions when fetching from crates.io (e.g. '0.6.0-alpha')
+    -B, --build                  Add crate as build dependency
+    -D, --dev                    Add crate as development dependency
+    -h, --help                   Prints help information
+        --no-default-features    Set `default-features = false` for the added dependency
+        --offline                Run without accessing the network
+        --optional               Add as an optional dependency (for use in features)
+    -q, --quiet                  Do not print any output in case of success
+    -s, --sort                   Keep dependencies sorted
+    -V, --version                Prints version information
+
+OPTIONS:
+        --git <uri>               Specify a git repository to download the crate from
+        --manifest-path <path>    Path to the manifest to add a dependency to
+        --path <path>             Specify the path the crate should be loaded from
+        --registry <registry>     Registry to use
+    -r, --rename <rename>         Rename a dependency in Cargo.toml, https://doc.rust-
+                                  lang.org/cargo/reference/specifying-
+                                  dependencies.html#renaming-dependencies-in-cargotoml Only works when
+                                  specifying a single dependency
+        --target <target>         Add as dependency to the given target platform
+        --upgrade <method>        Choose method of semantic version upgrade [default: default]  [possible values: none,
+                                  patch, minor, all, default]
```

Comment:
could you bring the explanations for `none`, `patch`, etc back?

Reply:
Sure. Should I add it to the readme only or also to the output of `cargo add -h` or `cargo add --help`?

Reply:
that would be a nice addition too, thanks

Reply:
I added it to the help output and updated the pull request.